Project.toml

```diff
@@ -1,13 +1,13 @@
 name = "FFMPEG"
 uuid = "c87230d0-a227-11e9-1b43-d7ebe4e7570a"
 authors = ["SimonDanisch <sdanisch@gmail.com>"]
-version = "0.4.1"
+version = "0.4.2"
 
 [deps]
 FFMPEG_jll = "b22a6f82-2f65-5046-a5b2-351ab43fb4e5"
 
 [compat]
-FFMPEG_jll = "4.3.1"
+FFMPEG_jll = "4.3.1, 6.1.1"
```

Comment:
Hmmm.. I'm not sure about adding this in a patch release. 

At least can we check VideoIO.jl tests before releasing this

Reply:
> Hmmm.. I'm not sure about adding this in a patch release.

I agree, that would also mean moving the julia bound to 1.6

> At least can we check VideoIO.jl tests before releasing this

By this do you mean testing locally?

Reply:
Maybe also move to v1.0.0 altogether, to avoid the mess of using only two version numbers, this package hasn't been touched in 3 years, looks moderately stable.

Reply:
> By this do you mean testing locally?

You could edit the VideoIO CI to dev this branch and test on a PR

Reply:
> Maybe also move to v1.0.0 altogether, to avoid the mess of using only two version numbers, this package hasn't been touched in 3 years, looks moderately stable.

I see there are these drone CI running on Julia 1.3 - 1.5, can I remove it safely?

Reply:
The Drone service is completely broken, just remove it.

Reply:
I also remove AppVeyor as the service in Julia is unmaintained and version still refers to 1.5